about/introduction.rst

```diff
@@ -17,6 +21,33 @@ This page gives a broad presentation of the engine and of the contents
 of this documentation, so that you know where to start if you are a beginner or
 where to look if you need info on a specific feature.
 
+Where to start
+--------------
+
+So you're looking to get into Godot Engine - great!
+See the **About the documentation** section below for more details on how the
+Godot documentation is structured, but if you're just wondering
+*Where should I start?*, here are some pointers, depending on your preferences:
+
+**I am a complete beginner to game development.** |br|
+Reading this documentation in order is a good way to learn!
+So read on, finish this page and head on to the first section:
+:ref:`Step by step <toc-learn-step_by_step>`.
+
+**I already have some experience or just want to jump right in!** |br|
+Skip some general introduction to the editor and start with
+:ref:`your first game <doc_your_first_game>`.
+
+**All this text is boring me, give me something to watch!** |br|
+The :ref:`Tutorials and resources <doc_community_tutorials>` page lists
+video tutorials contributed by the community.
+
+**I need some help, my project isn't working.** |br|
+Take a look at the :ref:`Community channels <doc_community_channels>`,
+especially the Godot Discord community, Q&A and IRC.
+
+Whichever way you choose to start with: have fun and may Godot be with you!
```

Comment:
Drop the first "with". May sound better: "However you choose to start"

Is "may Godot be with you" a thing now? :P